test/parallel/test-diagnostics-channel-module-require.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const dc = require('diagnostics_channel');
+
+const trace = dc.tracingChannel('module.require');
+const events = [];
+let lastEvent;
+
+function track(name) {
+  return (event) => {
+    // Verify every event after the first is the same object
+    if (events.length) {
+      assert.strictEqual(event, lastEvent);
+    }
+    lastEvent = event;
+
+    events.push({ name, ...event });
```

Comment:
```suggestion
    events.push(even});
```
according to docs `name` should be part of the event.
But maybe the doc is wrong and test is fine.